src/adapters/NoOpNetworkAdapter.js

```diff
@@ -21,6 +21,8 @@ class NoOpNetworkAdapter extends NafInterface {
 
   getMediaStream(clientId) { return Promise.reject("Interface method not implemented: getMediaStream")}
 
+  getTime() { return Date.now(); }
```

Comment:
I vote we call this something like `getServerTime`. Also the rest of the methods in noop adapter just throw a not implemented error, this should probably do the same.. That said we should probably come up with a nicer way to do this. I forget why we don't just have a base `Adapter` that is not registered but is subleased by all the adapters so that we can provide functionality like this as a base.

Reply:
I believe we got rid of the network adapters all extending some base adapter because of circular dependencies. Ex.

```